addons/spreadsheet/static/src/pivot/plugins/pivot_ui_plugin.js

```diff
@@ -510,6 +521,28 @@ export class PivotUIPlugin extends spreadsheet.UIPlugin {
             this.dataSources.add(dataSourceId, PivotDataSource, definition);
         }
     }
+
+    _getUnusedPivots() {
```

Comment:
`_usedValueDomains` doesn't really help does it ? It's only ever filled, the values inside of it are not removed if the pivot cell is deleted. And it's not like I can clear this cache at each cell update, because the pivot cells may not be re-evaluated.